### PR TITLE
github actions

### PR DESCRIPTION
**Pull request by delphiactual**
Need to update our secrets... @bhollis @sundevour 

~~`Settings` -> `Secrets`~~

~~https://github.com/marketplace/actions/install-ssh-key~~

### CODE REVIEW

**Comment by bhollis** [comment]
Added! Super interested to see how GitHub actions compares to Travis.

**Comment by delphiactual** [comment]
So this works, just need to make sure it doesnt make PR to DIM/d2ai unless merged.

**Comment by delphiactual** [comment]
I will write up another github action for Pull Requests, that makes sure all output files are current; and if not will commit directly to the PR

**Comment by delphiactual** [comment]
https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow

**Comment by delphiactual** [comment]
now to setup the cron to trigger another workflow when a new version is available

File: scripts/ci.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    echo =================End Changes To $1==================
+    echo
+}
+
+git config --global user.email "destinyitemmanager@gmail.com"
+git config --global user.name "DIM Config Bot"
+
+# Clone project
+#git clone git@github.com:DestinyItemManager/d2-additional-info.git --depth 1
+#cd d2-additional-info
```
Comment (delphiactual) [inline scripts/ci.sh:16]: not required as github actions does this automatically

File: scripts/ci.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    then exit 0
+fi
+
+# continue on to check for changes from current dim configs
+# we clone DIM, and add our new files
+cd ..
+hub clone DestinyItemManager/DIM --depth 1
+cd DIM
+find src/data/d2/* -delete
+cp -f ../d2-additional-info/output/* ./src/data/d2/
```
Comment (delphiactual) [inline scripts/ci.sh:62]: this might be wrong because of the need to no longer clone d2ai

**Pull request merged**